packages/trader/src/sass/add-or-manage.scss

```diff
@@ -1,18 +1,11 @@
 @import 'app/_common/base/index';
 
 .add-or-manage {
-    display: flex;
-    width: 100%;
-    flex-direction: column;
-    justify-content: space-between;
-    flex-grow: 1;
-    flex-shrink: 0;
+    flex-direction: row; /* fix for Safari */
```

Comment:
Will it cause any style issue for other browsers? If it's a specific fix the shouldn't we target it with a media query?

Reply:
no, it works across browsers